minos/common/database/database_pool.py

```diff
@@ -0,0 +1,26 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+
+import aiomisc
+import aiopg
+
+
+class AiopgPool(aiomisc.PoolBase):
```

Comment:
I think that `PostgresPool` or something similar would be a better name (when we use it, we don't care if it's based on `aiomisc` or another tool in this case).

Reply:
Done